src/views/layouts/_userinfo.php

```diff
@@ -9,19 +9,26 @@
  * @var Client $client
  */
 
-$client = Client::findOne(Yii::$app->user->identity->id);
+$client = Client::find()
+    ->where([
+        'id' => Yii::$app->user->identity->id,
+    ])
+    ->addSelect(['purses'])
+    ->withPurses()
```

Comment:
```suggestion
    ->withPurses()
```

Нужно чтобы withPurses() сам делал `->addSelect(['purses'])`

Reply:
поддерживаю